client.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function
+import logging
+
+import grpc
+
+import echo_pb2
+import echo_pb2_grpc
+
+
+def run():
+    # NOTE(gRPC Python Team): .close() is possible on a channel and should be
+    # used in circumstances in which the with statement does not fit the needs
+    # # of the code.
+
+    channel = grpc.insecure_channel('localhost:50051')
```

Comment:
What happens if the server runs on another machine? What happens if the server runs on a different port? Changing the code every time for different deployments is not a good practice. A good solution for this case would be to inject those values **as configuration**.

There are several ways to accomplish that, ordered by level of complexity (and features):

1. [argv](https://www.tutorialspoint.com/python/python_command_line_arguments.htm)
2. [argparse](https://docs.python.org/3/library/argparse.html)
3. [click](https://click.palletsprojects.com/en/8.0.x/)